UserService/Domain/User.cs

```diff
@@ -31,7 +29,6 @@ public User(
     }
 
     public string FirebaseId { get; set; }
-    public string Username { get; set; }
```

Comment:
Please make migration that will propagate this change to database.

You can do that in Package-Manager console with command
Add-Migration <MigrationName>